### ISSUE TITLE
Details about ImageNet experiments setting

### ISSUE THREAD

**Issue by licj15** [htqin/IR-Net#6]
@htqin Thank you for your quick response! I intended to ask this in this [issue](https://github.com/htqin/IR-Net/issues/5), but it was closed and this seems to be a new question, so I open a new issue for this question. Thank you a lot!

I notice in the yaml

```
    augmentation:
        input_size: 224
        test_resize: 256
        colorjitter: [0.2, 0.2, 0.2, 0.1] 
```
1. Does it mean you use 224224 for train, and upsample it to 256256 during test? If so, I noticed you didn't mention so in your paper, do you have any references for this? I was wondering if such techniques are also used in baselines.
2. Do you only use "colorjitter: [0.2, 0.2, 0.2, 0.1]" for augmentation? Do you use random crop or flip? Because I noticed you use such augmentation in CIFAR-10, not sure if you use any other augmentation beside "colorjitter: [0.2, 0.2, 0.2, 0.1]" for ImageNet.

Thank you! Your efforts and the sprit to open source will benefit the community a lot!

**Comment by Haleski47** [comment]
> @htqin Thank you for your quick response! I intended to ask this in this [issue](https://github.com/htqin/IR-Net/issues/5), but it was closed and this seems to be a new question, so I open a new issue for this question. Thank you a lot!
> 
> I notice in the yaml
> 
> ```
>     augmentation:
>         input_size: 224
>         test_resize: 256
>         colorjitter: [0.2, 0.2, 0.2, 0.1] 
> ```
> 
> 1. Does it mean you use 224224 for train, and upsample it to 256256 during test? If so, I noticed you didn't mention so in your paper, do you have any references for this? I was wondering if such techniques are also used in baselines.
> 2. Do you only use "colorjitter: [0.2, 0.2, 0.2, 0.1]" for augmentation? Do you use random crop or flip? Because I noticed you use such augmentation in CIFAR-10, not sure if you use any other augmentation beside "colorjitter: [0.2, 0.2, 0.2, 0.1]" for ImageNet.
> 
> Thank you! Your efforts and the sprit to open source will benefit the community a lot!

i'm also curious about the training details for imagenet dataset. i see the traing learning rate is 0.8 for warmup stage from the taml file. it's a bit crazy...

**Comment by htqin** [comment]
Thanks for your attention! @licj15 @Haleski47 
A1: We use the same augmentation setting on both full-precision and binary networks. These settings are mainly for full-precision networks and seem to improve the performance of baselines (including augmentation and warmup lr). Thank you for pointing out, we will claim our various settings more clearly in future work.
A2: [Q2: "if you use any other augmentation beside "colorjitter: [0.2, 0.2, 0.2, 0.1]" for ImageNet"] No, the networks for CIFAR-10 and imagenet are based on different projects, and we follow their setting of augmentation.
(ResNet CIFAR-10: https://github.com/akamaster/pytorch_resnet_cifar10)
